openlibrary/templates/books/RelatedWorksCarousel.html

```diff
@@ -12,12 +12,12 @@
             $ )
             $if author_keys:
                 $ query += ' -author_key:(%s)' % ' OR '.join(['%s' % a for a in author_keys])
-            $:macros.QueryCarousel(query=query, title=_("You might also like"), key="related-subjects-carousel", limit=12, user_lang_only=True)
+            $:macros.QueryCarousel(query=query, title=_("You might also like"), key="related-subjects-carousel", limit=12, user_lang_only=True, sort='')
```

Comment:
This is unrelated to this PR but @cdrini noticed a bug where related works was sorted by new instead of relevance